axelor-bank-payment/src/main/resources/views/BankReconciliationLine.xml

```diff
@@ -47,6 +47,41 @@
     <field name="isSelectedBankReconciliation" hidden="true"/>
   </grid>
 
+  <grid name="bank-reconciliation-line-move-line-grid" title="Bank reconciliation Lines"
+    model="com.axelor.apps.bankpayment.db.BankReconciliationLine" canDelete="false" editable="false"
+    canNew="false">
+    <field name="postedNbr"/>
+    <field name="confidenceIndex" readonly="true">
+      <hilite if="confidenceIndex == 1" color="green"/>
+      <hilite if="confidenceIndex == 2" color="orange"/>
+      <hilite if="confidenceIndex == 3" color="red"/>
+    </field>
+    <field name="bankStatementQuery"/>
+    <field name="effectDate"/>
+    <field name="name"/>
+    <field name="reference"/>
+    <field name="debit" aggregate="sum">
+      <hilite if="debit > 0" color="red"/>
+    </field>
+    <field name="credit" aggregate="sum">
+      <hilite if="credit > 0" color="green"/>
+    </field>
+    <field name="bankStatementLine.amountRemainToReconcile" aggregate="sum"/>
+    <field name="typeSelect" onChange="action-bank-reconciliation-line-record-set-account"/>
+    <field name="account" canEdit="false"
+      onSelect="action-bank-reconciliation-line-method-account-domain" form-view="account-form"
+      grid-view="account-grid"/>
+    <field name="partner" canEdit="false"
+      onSelect="action-bank-payment-attrs-bank-reconciliation-line-partner-domain"
```

Comment:
onSelect and onChange event can be removed. All columns should be displayed in readonly